lib/pp.ml

```diff
@@ -351,6 +351,11 @@ let pr_nth n =
   in
   int n ++ str s
 
+let pluralize word l : std_ppcmds =
+  if List.length l > 1 || List.length l == 0
```

Comment:
This is very minor, but why not flip this around?  `if List.length == 1 then str word else ....`